ROOTFS/etc/sage-utils/dhcp-pxe/nfs/etc/rc.local

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+#initializes sd card on rpi
+
+SDDIR="/dev/mmcblk0"
+SDPARTDIR="/dev/mmcblk0p1"
+
+if [ -b "$SDDIR" ]; then
+  echo "SD Card found continuing..." | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` > /etc/rc.local.logs
+  mkdir -p /media/scratch/ | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` >> /etc/rc.local.logs
+  if [ -b "$SDPARTDIR" ]; then
+        echo "Mounting SD CARD Scratch Space" | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` >> /etc/rc.local.logs
+        mkfs -t ext4 /dev/mmcblk0p1 | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` >> /etc/rc.local.logs
+        mount /dev/mmcblk0p1 /media/scratch | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` >> /etc/rc.local.logs
```

Comment:
I would make a function for the mounting and call it here and then on line 28 instead of duplicating this code.